Updates modular installation commands and limitations (#5040)
isaac-sim/IsaacLab: docs/source/setup/installation/include/src_build_isaaclab.rst

            isaaclab.bat --install :: or "isaaclab.bat -i"

   By default, the above will install **all** Isaac Lab submodules (under ``source/isaaclab``).
   To install only specific Isaac Lab submodules, pass a comma-separated list of submodule names. The available
   Isaac Lab submodules are: ``assets``, ``contrib``, ``mimic``, ``newton``, ``ov``, ``physx``, ``rl``, ``tasks``,
   ``teleop``, ``visualizers``. Available RL frameworks are: ``rl_games``, ``rsl_rl``, ``sb3``, ``skrl``, ``robomimic``.

>    For example, to install only the minimally required submodules:

Reviewer: I think it's confusing when used in combination with `the minimally required`

>    For example, to install only the minimally required submodules:

Reviewer: ```suggestion
   For example, to install a small subset of submodules:
```

         .. code:: bash

>             ./isaaclab.sh --install physx,newton,assets,rl[rsl_rl],tasks,ov  # or "./isaaclab.sh -i physx,newton,assets,rl[rsl_rl],tasks,ov"

Reviewer: Do we want to list physx here?

Reviewer: see above comment

Author: ya this is needed unfortunately because of the way the cfg scripts are setup :(

      .. tab-item:: :icon:`fa-brands fa-windows` Windows
         :sync: windows

         .. code:: batch

            isaaclab.bat --install physx,newton,assets,rl[rsl_rl],tasks,ov :: or "isaaclab.bat -i physx,newton,assets,rl[rsl_rl],tasks,ov"

   To install specific visualizer, pass a comma-separated list of supported visualizers,

>    or ``all`` to install all available options: ``newton``, ``rerun``, ``viser``, ``kit``.

Reviewer: ```suggestion
   To install specific visualizer, pass a comma-separated list of supported visualizers,
   or ``all`` to install all available options: ``newton``, ``rerun``, ``viser``, ``kit``. Note when following the 
   default installation, all visualizers are installed by default.
```
